### ISSUE TITLE
Remove class

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/spc-parser#30]
I don't think a class is necessary in this project because we don't have methods on an instance. Those are simple objects.

For example:

https://github.com/santimirandarp/spc-parser/blob/8bf7dc0412da6859e268b5fb9ea9c3f3bd6476a3/src/utility.ts#L49-L69

**Comment by santimirandarp** [comment]
Yes, I'll fix it. I'd just comment when the function returns an object, and does a very simple process, the code ends up easier to read in a class because the interface is defined right there, and there is no need for  something like `let ourObject = { }` and at the end `return ourObject as ThisType`.

**Comment by santimirandarp** [comment]
i think this was solved
